examples/kubernetes/access_points/README.md

```diff
@@ -0,0 +1,78 @@
+## EFS Access Points
```

Comment:
I think certain IAM permission is required in order to mount access point, What's the IAM permission used in this case? And how is the IAM permission assigned? Using instance profile or IAM for SA?

Reply:
The driver (under the DaemonSet) is already running with godlike powers; I don't think anything additional is required. At least, I haven't run into any issues. But in any case I would welcome someone confirming my findings by walking through the doc :)

Reply:
Let’s confirm on that and document here. As a minimum permission, this driver doesn’t require any IAM permission for it to work

Reply:
We confirmed another way that -- by default -- no IAM permissions are required. I've noted that it's possible to set up additional policies.

Reply:
LGTM, thx for confirming!